creator-node/src/components/contentBlacklist/contentBlacklistComponentService.ts

```diff
@@ -3,12 +3,29 @@ const models = require('../../models')
 
 const types = models.ContentBlacklist.Types
 
-const getAllTrackIds = async () => {
+interface IBlacklistedContent {
+  trackIds: Promise<any>;
```

Comment:
As mentioned in my other comment, `trackIds` is probably `Promise<string[]>`. `userIds` though I think is actually `Promise<number[]>`.

UserId Example: https://discoveryprovider3.audius.co/users?handle=jonaylor89
TrackId Example: https://discoveryprovider3.audius.co/v1/tracks/jlg9G

Reply:
Ah yes, looks like `Promise<number[]>` to me as well for `userIds`. done and done.